python/activator/visit.py

```diff
@@ -54,9 +54,12 @@ class Dome(enum.IntEnum):
         OPEN = 2
         EITHER = 3
 
-    salIndex: int               # this maps to an instrument
+    # script queue that generated the event. One queue usually runs one telescope, but they can switch
+    salIndex: int
     scriptSalIndex: int
-    groupId: str                # observatory-specific ID; not the same as visit number
+    # Observatory-specific ID. Same as Butler's group_name, not the same as
+    # Butler/RubinTV's group_id or visit number
```

Comment:
I'm not sure we need to mention RubinTV because (1) RubinTV just shows whatever from Butler with a thin name mapping (2) RubinTV might change and that can be confusing to others reading the comment in the future.   It's actually easy to get RubinTV to call the column differently [(slack thread here);](https://lsstc.slack.com/archives/C04SW09ECCE/p1699981196663919) I was thinking to PR that, but then think that can add even more confusion...